src/Pages/namjunpark/Main/Components/Feed.js

```diff
@@ -0,0 +1,149 @@
+import React, { Component } from "react";
+
+class Feed extends Component {
+  constructor() {
+    super();
+    this.state = {
+      id: 0,
+      comment: "",
+      comments: [],
+    };
+  }
+
+  handleComment = (e) => {
+    const { value } = e.target;
+    this.setState({
+      comment: value,
+    });
+  };
+
+  addComment = (e) => {
+    const { id, comment, comments } = this.state;
+    if (comment) {
+      this.setState({
+        id: id + 1,
+        comments: [
+          ...comments,
+          {
+            id: id,
```

Comment:
```suggestion
            id,
```
object에서 key와 value가 같을 때는 줄일 수 있어요!